android/examples/run-first-test/features/terrain.py

```diff
@@ -0,0 +1,24 @@
+from lettuce import before, after, world
+from browserstack.local import Local
+from appium import webdriver
+import os, json
+
+config_file_path = os.path.join(os.path.dirname(__file__), '..', "config.json")
+
```

Comment:
can we print the config file path?

Reply:
I removed the print statements for now as they were printing twice. We can take it up in next iteration.